smoke/src/main/java/com/adobe/cq/cloud/testing/it/smoke/PersistedQueryIT.java

```diff
@@ -0,0 +1,51 @@
+/*************************************************************************
+ * ADOBE CONFIDENTIAL
+ * ___________________
+ *
+ * Copyright 2021 Adobe
+ * All Rights Reserved.
+ *
+ * NOTICE: All information contained herein is, and remains
+ * the property of Adobe and its suppliers, if any. The intellectual
+ * and technical concepts contained herein are proprietary to Adobe
+ * and its suppliers and are protected by all applicable intellectual
+ * property laws, including trade secret and copyright laws.
+ * Dissemination of this information or reproduction of this material
+ * is strictly forbidden unless prior written permission is obtained
+ * from Adobe.
+ **************************************************************************/
+package com.adobe.cq.cloud.testing.it.smoke;
+
+import com.adobe.cq.testing.client.CQClient;
+import com.adobe.cq.testing.junit.rules.CQPublishClassRule;
+import com.adobe.cq.testing.junit.rules.CQRule;
+import org.apache.sling.testing.clients.ClientException;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+
+public class PersistedQueryIT {
+
+    @ClassRule
+    public static final CQPublishClassRule cqBaseClassRule = new CQPublishClassRule();
+    static CQClient adminPublish;
+
+    @Rule
+    public CQRule cqBaseRule = new CQRule(cqBaseClassRule.publishRule);
+
+    @BeforeClass
+    public static void beforeClass() {
+        adminPublish = cqBaseClassRule.publishRule.getAdminClient(CQClient.class);
```

Comment:
I think you need the equivalent of an anonymous user that hits the dispatcher. @dulvac or @volteanu could give you some more hints about which client would be best or how to construct one that would serve the purpose for this PR.

Reply:
you can get an anonymous user by passing `null` to user/pass: 
```suggestion
        anonymousPublish = cqBaseClassRule.publishRule.getClient(CQClient.class, null, null);
```

Reply:
fixed